src/Sulu/Bundle/PageBundle/Resources/translations/admin.en.json

```diff
@@ -61,5 +61,18 @@
     "sulu_page.target_groups_description": "The block will only be visible for the selected target groups.",
     "sulu_page.add_schedule": "Add schedule",
     "sulu_page.schedules_label": "Assign time schedule",
-    "sulu_page.schedules_description": "The block will only be visible if at least one condition is valid."
+    "sulu_page.schedules_description": "The block will only be visible if at least one condition is valid.",
+    "sulu_activity.text.pages.created": "{userFullName} has created the page {resourceTitle}",
+    "sulu_activity.text.pages.modified": "{userFullName} has changed the page {resourceTitle}",
+    "sulu_activity.text.pages.removed": "{userFullName} has removed the page {resourceTitle}",
+    "sulu_activity.text.pages.published": "{userFullName} has published the page {resourceTitle}",
+    "sulu_activity.text.pages.unpublished": "{userFullName} has unpublished the page {resourceTitle}",
+    "sulu_activity.text.pages.draft_removed": "{userFullName} has discarded the draft of the page {resourceTitle}",
+    "sulu_activity.text.pages.copied": "{userFullName} has copied the page {resourceTitle}",
+    "sulu_activity.text.pages.moved": "{userFullName} has moved the page {resourceTitle} from \"{context.previousParentTitle}\" to \"{context.newParentTitle}\"",
```

Comment:
i think you mean that we cannot easily check if the `context.previousParentTitleLocale` is different to the `resourceLocale`, right? i think it would be okay to just always/never display them 🙂

Reply:
not different to the `resourceLocale`, different to the locale of the user ... I would still prefer something like we did in the `ActivityController` for the `resourceTitleLocale`, if there is an "easy" way to do this ...

Reply:
Maybe we could override all `context.*Title`, if both `context.*Title` and `context.*TitleLocale` is given? I know this is kind of hacky, but I could live with that ...

Reply:
i guess we could check if there is a translation parameter with the same name and a `Locale` suffix for each of the translations parameters to generalize it. 

i think i would not add the quotes like we do in the `resourceTitle` at the moment, but just append the locale like this `parameter value [locale]`. i think its a bit confusing at the moment that the `resourceTitle` already contains quotes but all other parameters do not

Reply:
But the result should be `"Page A" [EN]` and this is not possible if we don't add the quotes already to the translation parameter

Reply:
i think it would be okay to display `"Page A [EN]"` instead of `"Page A" [EN]` in the UI if it makes things easier/more consistent to use🙂

Reply:
Then I think we should change that for the `resourceTitle` too and remove the quotes from the translation parameter

Reply:
yes would be okay for me 🙂